submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/data/web/SubmissionDraftResource.kt

```diff
@@ -85,6 +88,7 @@ internal class SubmissionDraftResource(
         @BioUser user: SecurityUser,
         onBehalfRequest: OnBehalfRequest?,
         @RequestParam(FILE_MODE, defaultValue = "COPY") mode: FileMode,
+        @RequestParam(PREFERRED_SOURCE, defaultValue = "USER_SPACE") preferredSource: PreferredSource,
```

Comment:
1. This and every submission method is getting super loaded with all this parameters, I think we need to introduce class here for example annotated with model attribute.